docs/src/pages/components/data-grid/style/StylingCellsGrid.tsx

```diff
@@ -1,17 +1,19 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import { GridColumns, DataGrid, GridCellClassParams } from '@material-ui/data-grid';
+import { GridColumns, DataGrid, GridCellParams } from '@material-ui/data-grid';
 import { makeStyles } from '@material-ui/styles';
 
 const columns: GridColumns = [
   {
     field: 'name',
     cellClassName: 'super-app-theme--cell',
+    width: 140,
```

Comment:
I'm not sure that we should care about this now. I think that it would be better to handle it all in #1712.

Reply:
I reverted this change. Only did it because I was already touching these files.